src/HookHandler/PageApprovalParserHooks.php

```diff
@@ -2,13 +2,13 @@
 
 declare( strict_types = 1 );
 
-namespace ProfessionalWiki\PageApprovals\EntryPoints;
+namespace ProfessionalWiki\PageApprovals\HookHandler;
 
 use OutputPage;
 use ParserOutput;
 use ProfessionalWiki\PageApprovals\PageApprovals;
 
-class PageApprovalsHooks {
+class PageApprovalParserHooks {
```

Comment:
This class should be combined with yours. We will basically have one class for hook methods, and they will call other services as necessary. We don't need to create separate hook clases.